conda_mirror/conda_mirror.py

```diff
@@ -26,6 +26,7 @@
                      'win-64',
                      'win-32']
 
+MINIMUM_FREE_SPACE_MB = 1000
```

Comment:
It would help to make variable a command line argument instead so that users can decide what minimum means on their infrastructure and testing becomes more feasible.

Reply:
A command line parameter with that default if not specified sounds good to me.